### PR TITLE
Implement Oracle

### PR DESCRIPTION
**Pull request by tanim0la**
Description
-----------

This PR implements the ChainLink Oracle for the Liquid Protocol, which fetches the price of a token in USD.

Motivation and Context
----------------------

This component of the Liquid Protocol get the current prices of various tokens in USD, enabling accurate calculation of the collateral factor percentage.

How Has This Been Tested?
-------------------------

Screenshots (if appropriate, particularly for frontend repos)
----------------------------
nil

Types of changes
----------------

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

Checklist
---------

- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful. (Please rebase, removing useless commits if not)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] My change requires change(s) to the environment.
- [ ] Communication sent to affected teams

- [ ] Other information.

### CODE REVIEW

File: src/protocols/common/constant.sol [hunk 1]
```diff
@@ -11,6 +11,13 @@ abstract contract Constants {
     // Wrapped Ether
     address internal constant WETH = 0x4200000000000000000000000000000000000006;
 
+    // ChainLink Price Feeds contract
+    address internal constant SEQUENCER_UPTIME_FEED = 0xBCF85224fc0756B9Fa45aA7892530B47e10b6433; // base mainnet
+    address internal constant CBBTC_USD = 0x07DA0E54543a844a80ABE69c8A12F22B3aA59f9D;
+    address internal constant DAI_USD = 0x591e79239a7d679378eC8c847e5038150364C78F;
+    address internal constant ETH_USD = 0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70;
+    address internal constant USDC_USD = 0x7e860098F58bBFC8648a4311b374B1D669a2bc6B;
+
```
Comment (njokuScript) [inline src/protocols/common/constant.sol:20]: can you do usdc to any other token on chainlink? say usdc => aero?

Comment (njokuScript) [reply src/protocols/common/constant.sol:20]: Or we're being specific about the assets on strategies we can allow? so on the UI, you can't create a strategy that involves any asset outside CBBTC, DAI, USDC, and ETH?

Comment (tanim0la) [reply src/protocols/common/constant.sol:20]: not directly. but i can come up with an helper function for it

Comment (njokuScript) [reply src/protocols/common/constant.sol:20]: that makes sense then. can we have a utility contract to get supported assets and protocols, and it'll return the associated  addresses.

**Review by njokuScript — APPROVED** [review]

**Pull request merged**